src/styles/blocks/header.scss

```diff
@@ -0,0 +1,12 @@
+.header {
+  display: flex;
+  align-items: center;
+  background-color: #fff;
+  justify-content: space-between;
```

Comment:
Box-shadow property for the header is missing. Pay attention to the design
![image](https://github.com/mate-academy/layout_catalog/assets/96409563/b8686eda-a84c-4b0a-883a-30c21bcd6d22)
![image](https://github.com/mate-academy/layout_catalog/assets/96409563/0d47db45-a4cb-41a3-ad33-ac7d326be9bf)
![image](https://github.com/mate-academy/layout_catalog/assets/96409563/76aadabb-b3c3-4c53-be91-c3af5727c1ab)